<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="C02WR0FCHV2R" name="Command line test" tests="5" failures="2" timestamp="21 Sep 2019 07:11:08 GMT" time="4.334" errors="0">
  <testcase name="testApplicationResponse" time="0.003" classname="com.main.TestApplication">
    <failure type="java.lang.AssertionError" message="Value for Key Path is not matching in application response. expected [Invalid value] but found [/Business-farming-industry/Carbon-credits]">
      <![CDATA[java.lang.AssertionError: Value for Key Path is not matching in application response. expected [Invalid value] but found [/Business-farming-industry/Carbon-credits]
	at com.main.TestApplication.testApplicationResponse(TestApplication.java:126)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testApplicationResponse -->
  <testcase name="testApplicationResponse" time="0.006" classname="com.main.TestApplication"/>
  <testcase name="testApplicationResponse" time="0.001" classname="com.main.TestApplication"/>
  <testcase name="testApplicationResponse" time="0.001" classname="com.main.TestApplication">
    <failure type="java.lang.AssertionError" message="Key Invalid_key is not available in application response expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Key Invalid_key is not available in application response expected [true] but found [false]
	at com.main.TestApplication.testApplicationResponse(TestApplication.java:85)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testApplicationResponse -->
  <testcase name="testApplicationResponse" time="0.001" classname="com.main.TestApplication"/>
</testsuite> <!-- Command line test -->
